config/storage.yml

```diff
@@ -6,13 +6,12 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
-# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+amazon:
+  service: S3
+  access_key_id: <%= ENV['S3_ACCESS_KEY_ID'] %>
+  secret_access_key: <%= ENV['S3_SECRET_ACCESS_KEY_ID'] %>
```

Comment:
dlaczego nie rails credentials?

Reply:
lepiej jednak jak envy leca z kodem, bo inaczej bedziesz zmuszony finalnie samemu nimi zarzadzac, a to jest bol

Reply:
zrobiłem tak ja sugerowała dokumentacja Heroku: https://devcenter.heroku.com/articles/s3, w ENV też trzymam hasła na codzień - pozmieniam na rails credentials - będę miał porównianie :)